### DISCUSSION TITLE
Updating HW3-Alter results in black screen

### DISCUSSION THREAD

**Discussion by DrAzathoth** [sanni/cartreader#961] [Need help]
### OSCR firmware version

12.4

### OSCR hardware version

HW3 Alter

### Attach OSCR_LOG.txt file here

N/A

### System used

all/none/doesn't matter

### Describe the issue.

Perhaps I did something wrong, but flashing my 3-Alter with the latest update (13.1) results in a black screen. The Arduino still lights up, but otherwise, the device is unresponsive to the point of receiving no feedback from the serial monitor. If this issue is more appropriate for the discussions forum, let me know and I'll move it there.

**Comment by Ancyker** [comment]
What options did you change in `Config.h`?

**Reply by DrAzathoth** [reply]
I set it to HW3 and commented out the NES and SMS cores that were automatically enabled.

I can attach that file for reference if you want.

**Reply by Ancyker** [reply]
Did you enable the clock generator or RTC (if you have them)? I know if you enable RTC and don't have it the display will remain blank.

**Reply by SAVE-THE-HERO** [reply]
If that doesn't work, please contact me at my email.
savetheherobuilders@gmail.com

**Reply by DrAzathoth** [reply]
> Did you enable the clock generator or RTC (if you have them)? I know if you enable RTC and don't have it the display will remain blank.

I did not enable either because it said it would automatically be enabled for HW3. Just in case, I tried enabling both right now and the screen is still black.

**Reply by DrAzathoth** [reply]
> If that doesn't work, please contact me at my email. [savetheherobuilders@gmail.com](mailto:savetheherobuilders@gmail.com)

I'll try reaching out to you when I have the chance, thanks!

**Comment by Ancyker** [comment]
@SAVE-THE-HERO

**Reply by SAVE-THE-HERO** [reply]
THX

**Comment by DrAzathoth** [comment]
I resolved the issue. By default, a sample sketch was loaded by the program rather than the OSCR sketch. By loading the correct sketch I was able to flash the update without issue.
